config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_apikeyskeys.apikeys.cnrm.cloud.google.com.yaml

```diff
@@ -5,10 +5,10 @@ metadata:
     cnrm.cloud.google.com/version: 0.0.0-dev
   creationTimestamp: null
   labels:
+    cnrm.cloud.google.com/directcrd: "true"
```

Comment:
Nit: `cnrm.cloud.google.com/direct` (it's the controller that's direct, not really the CRD).  Or it could just the default if we don't specify tf2crd or dcl2crd ?

Reply:
I'll change this to "cnrm.cloud.google.com/direct".  And +1 to make the direct as default eventually.

Reply:
I rethink about this. I think we can be more "direct aggressive" here (and we should be good if we do):
- no CRD labels. By default we expect a direct controller to reconcile the resource. To use TF/DCL controller, the CRD has to have the corresponding label.
- A direct controller needs to be registered under direct dir. Otherwise, without non-label CRD will return error on running time. WDYT?